config/s3-config.json

```diff
@@ -1 +1 @@
-{ "accessKeyId": "AKIAVMCSKOEDO3RW3KWV", "secretAccessKey": "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"}
\ No newline at end of file
+{ "accessKeyId": "**********", "secretAccessKey": "*****"}
```

Comment:
This file should not be committed to the github repo. Credentials are handled automatically by the running AWS Lambda in production. You can use this in local development (or better, use `aws configure` CLI) but it should not be in git.

I've searched our AWS account for the accessKey that was committed and have not been able to find it. This leads me to believe it's from another AWS account. **This key MUST be revoked** as it was committed to a public repo and anyone can see the history.

Reply:
I intentionally left it with * because they cannot be uploaded on Github, the first AWS account in question was already revoked. PLease let me know how I can update the codes to refer to the AWS lambda in production for s3.